src/macros.rs

```diff
@@ -86,10 +87,30 @@ macro_rules! core_uri {
 #[macro_export]
 macro_rules! config_uri {
     ($path:expr) => {{
-        format!("{}/{}", $crate::config::base_config_service_url(), $path)
+        format!("{}{}", $crate::config::base_config_service_url(), $path)
     }};
     ($path:literal, $($arg:tt)*) => {{
         let full_path: String = format!($path, $($arg)*);
         config_uri!(full_path)
     }};
 }
+
+#[test]
```

Comment:
We normally put our tests in separate test files

Reply:
Ah yes. I was discussing with @jpalvarezl regarding this because the `macros` live in the root `src` dir. I thought about moving to a different folder 🤔 What's your recommendation here?

Reply:
No strong opinion, but it should be consistent in our project

Reply:
We could move them to `/utils/tests/macros.rs`

Reply:
Yes I can do that. For me it was just that the macros are on `/src/macros.rs/` and not under `src/utils`

Reply:
Moved in 2b21092